cmd/kola/options.go

```diff
@@ -51,6 +53,7 @@ func init() {
 	root.PersistentFlags().IntVarP(&kola.TestParallelism, "parallel", "j", 1, "number of tests to run in parallel")
 	sv(&kola.TAPFile, "tapfile", "", "file to write TAP results to")
 	sv(&kola.Options.BaseName, "basename", "kola", "Cluster name prefix")
+	ss("debug-systemd-units", []string{}, "full-unit-names.service to enable SYSTEMD_LOG_LEVEL=debug on. Specify multiple times for multiple units.")
```

Comment:
Each instance of the option will address a single unit, so the option name shouldn't be plural.